src/Coordination/KeeperDispatcher.cpp

```diff
@@ -310,20 +312,23 @@ void KeeperDispatcher::shutdown()
         if (server)
             server->shutdown();
 
-        KeeperStorage::RequestForSession request_for_session;
-
-        /// Set session expired for all pending requests
-        while (requests_queue->tryPop(request_for_session))
+        if (requests_queue)
```

Comment:
I'd prefer to write `while (requests_queue && requests_queue->tryPop(request_for_session))` below to have smaller indentation level, if one extra check in loop is not overhead.